lochness/box/__init__.py

```diff
@@ -296,7 +318,7 @@ def sync_module(Lochness: 'lochness.config',
         bx_base_obj = get_box_object_based_on_name(
                 client, bx_base, '0')
 
-        if bx_base_obj == None:
```

Comment:
This conidtion `bx_base_obj == None` didn't work before right?

Reply:
A shorthand for `if bx_base_obj is None:` is `if bx_base_obj`.

Reply:
>This conidtion bx_base_obj == None didn't work before right?
This worked before. I changed this line to follow PEP8.

Reply:
>A shorthand for if bx_base_obj is None: is if bx_base_obj.
really?